src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/Mock/MockConnection.cs

```diff
@@ -75,10 +75,10 @@ private static IPEndPoint GetIPEndPoint(EndPoint endPoint)
                     return new IPEndPoint(IPAddress.Loopback, dnsEndPoint.Port);
                 }
 
-                throw new InvalidOperationException($"invalid DNS name {dnsEndPoint.Host}");
+                throw new InvalidOperationException(SR.Format(SR.net_quic_invalid_dns, dnsEndPoint.Host));
```

Comment:
These Mock* types are going to go away before shipping.  I think it's probably not worth adding these as resources only to then remove them. @geoffkizer?

Reply:
Yeah, don't bother here.